Miniforge3/construct.yaml

```diff
@@ -1,4 +1,5 @@
 {% set version = os.environ.get("MINIFORGE_VERSION", "4.9.2-0") %}
+{% set mamba_version = os.environ.get("MAMBA_VERSION", "0.7.3-0") %}
```

Comment:
Can you remove this? This env variable is not set by anything

Reply:
Done